<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QReportDialog</class>
 <widget class="QWidget" name="QReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>30</number>
   </property>
   <item>
    <widget class="QTableWidget" name="userTableWidget">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Dexa</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Retinal</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ultrasound</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="smtpServerComboBox">
       <item>
        <property name="text">
         <string>Gmail</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>McMaster</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>email:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="sendPushButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pwdLabel">
       <property name="text">
        <string>passwd:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
